### PR TITLE
fix #58355 bitlinks.pw

### LINKED ISSUE #58355: bitlinks.pw
### Issue URL (Anti Adblock Script)
 [https://bitlinks.pw/sh/0oaH?v=ZTA4YXpEd1JtSzg5LjI4LjExMy44MEAwb2FIZTA4YXpEd1JtSw](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fbitlinks.pw%2Fsh%2F0oaH%3Fv%3DZTA4YXpEd1JtSzg5LjI4LjExMy44MEAwb2FIZTA4YXpEd1JtSw)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (2880), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Spanish/Portuguese, <br/>ABPindo

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#58355

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -447,6 +447,9 @@ smallseotools.com#%#//scriptlet("set-constant", "isAdBlockActive", "false")
 @@||smallseotools.com^$generichide,badfilter
 ! distro.tv - antiadblock
 ||distro.tv/js/blocker.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/58355
+@@||faupto.com/freeltc/libs/show_ads.js
+faupto.com#%#//scriptlet("set-constant", "show_ads_gr8_lite", "true")
```
Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt:452]: Is it related to this issue?

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -459,11 +462,9 @@ wallpapershome.com###welcome.adblock
 @@||faucetcrypto.com/ads^
 ! https://forum.adguard.com/index.php?threads/38814/
 hentaihaven.xxx#@#.adBanner
-! https://github.com/AdguardTeam/AdguardFilters/issues/57744
-bitlinks.pw#$#img[id^="ads-"] { position: absolute!important; left: -3000px!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/58355
```
Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt:465]: What about Safari?

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/antiadblock.txt:452]: Yes, after captcha sometimes we can see adblock detection in the iframe loaded from faupto.com

File: EnglishFilter/sections/antiadblock.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/antiadblock.txt:465]: I suppose that the scriptlet will work in Safari, isn't it?

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (krystian3w) [reply EnglishFilter/sections/antiadblock.txt:452]: I thought you had to fix everything else that got caught - not only reported trouble.

File: EnglishFilter/sections/antiadblock.txt (hunk 2 above)
Comment (krystian3w) [reply EnglishFilter/sections/antiadblock.txt:465]: Will work in 1.7.5 - changelog aprove use `set-constant`.

**Review by krystian3w — APPROVED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 2 above)
Comment (krystian3w) [reply EnglishFilter/sections/antiadblock.txt:465]: Will work in `v1.6.4+` - changelog approve use `set-constant`: 
[AdguardTeam/AdGuardForSafari/releases/tag/v1.6.4](https://github.com/AdguardTeam/AdGuardForSafari/releases/tag/v1.6.4) + https://github.com/AdguardTeam/AdGuardForSafari/issues/237

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/antiadblock.txt:452]: got it

File: EnglishFilter/sections/antiadblock.txt (hunk 2 above)
Comment (Alex-302) [reply EnglishFilter/sections/antiadblock.txt:465]: > I suppose that the scriptlet will work in Safari, isn't it?

try JS for macos Safari only.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**